templates/pages/loggeduser.html

```diff
@@ -0,0 +1,305 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Projects{% endblock %}
+
+{% block content%}
+	
+ <nav class="navbar navbar-default" role="navigation" style="margin-bottom: 0">
```

Comment:
Maybe a bad idea, but I tried to simulate a situation when someone is logged in. The logged in state will probably somehow transform the navigation bar... I duplicated only to show and to make possible the toggle button to work on smaller screens. I don't know how this will be resolved in Python dynamically, so i presented a whole scenario.